docs/storage-and-interfaces.md

```diff
@@ -0,0 +1,59 @@
+> [!NOTE]
+> In large parts copied from [the storage-and-interfaces repo](https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces/blob/86d971294eb371189331c4898befe6dca8604b4b/README.md)
+To be updated and specialized for the ejprd use case.
+# Set up starter-kit-storage-and-interfaces in demo mode
+
+1. Download the repo https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces/
+1. Go into the repo's root folder `cd starter-kit-storage-and-interfaces`
+1. Run the following commands, to sure that all configuration files are in place.
+    ```shell
+    cp ./config/config.yaml.example ./config/config.yaml
+    cp ./config/iss.json.example ./config/iss.json
+    cp ./.env.example ./.env
+    ```
+
+1. To start the stack:
+```shell
+docker compose -f docker-compose-demo.yml up -d
+```
+
+The services will run in demo mode, with a mock-oidc in place of LS-AAI.
+Test data is loaded, loaded from here  https://github.com/ga4gh/htsget-refserver/tree/main/data/gcp/gatk-test-data/wgs_bam.
+
+After deployment is done, follow the instructions below to test that the demo worked as expected.
+
```

Comment:
Maybe we could add something about the way to take down the demo services, so that the volumes and the containers are removed, since the demo "works only once" and some parts fail if you try to restarted (such as the data loader). So maybe something like 
```suggestion
In order to restart the services, first take everything down and remove the volumes using
docker compose -f docker-compose-demo.yml down -v --remove-orphans
```